### PR TITLE
add reset password toggle

### LINKED ISSUE #1421: Add toggle to show/hide password on reset password modal
the modal looks like this at the moment
image

lets add a small checkbox below the text inputs like below
image

the checkbox is checked by default, and unchecking it would make the passwords hidden like so:
image

### PR DESCRIPTION
**Pull request by kevintsoii**
resolves #1421 

image
image

### CODE REVIEW

File: src/Pages/Profile/MemberView/ChangePassword.js [hunk 1]
```diff
@@ -52,13 +58,26 @@ export default function ChangePasswordModal(props) {
...
         </div>
+
+        <div className="form-control mt-1">
+          <label className="label cursor-pointer">
+            <span className="label-text">Show Password</span>
+            <input
+              type="checkbox"
+              className="toggle"
+              checked={showPassword}
+              onChange={togglePassword}
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/ChangePassword.js]: then you dont need the helper function
```suggestion
              onChange={() => setShowPassword(!showPassword)}
```

File: src/Pages/Profile/MemberView/ChangePassword.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ export default function ChangePasswordModal(props) {
   const { bannerCallback = (message, color) => { }, confirmClassAddons } = props;
   const [password, setPassword] = useState('');
   const [confirmPassword, setConfirmPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/ChangePassword.js:8]: by default we should show it?

File: src/Pages/Profile/MemberView/ChangePassword.js (hunk 1 above)
Comment (kevintsoii) [reply src/Pages/Profile/MemberView/ChangePassword.js]: done

File: src/Pages/Profile/MemberView/ChangePassword.js (hunk 2 above)
Comment (kevintsoii) [reply src/Pages/Profile/MemberView/ChangePassword.js:8]: I think its usually hidden by default on most sites I remember

**Review by evanugarte — APPROVED** [review]

**Review by pnavab — APPROVED** [review]
nice good

**Pull request merged**